lib/Parser/parse.cpp

```diff
@@ -7236,7 +7237,7 @@ ParseNodePtr Parser::ParseExpr(int oplMin,
                 //
                 // Otherwise it is an error for a yield to appear in the context of a higher level
```

Comment:
Oh I meant to remove that comment since it was incorrect (from copy paste). Thanks, I'll remove it.